docs/todomvc.md

```diff
@@ -0,0 +1,129 @@
+# TodoMVC step by step example
+
+This [example](/examples/todomvc) will show you how to deploy a famous [TodoMVC](https://todomvc.com/) website using Kusk Gateway.
+
+Please refer to the [installation](/docs/installation.md) guide on how to get Kusk Gateway Manager installed into your Kubernetes cluster.
+
+We chose the [TodoBackend](http://www.todobackend.com/) implementation for an example. The website consists of a Go-powered
+[backend](/examples/todomvc/backend) (with an OpenAPI [specification](/examples/todomvc/todospec.yaml) provided) and a NodeJS [frontend](/examples/todomvc/frontend).
+
+In order to let the website communicate with the backend, one would need to properly configure [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+Luckily, Kusk Gateway Manager allows you to do that right in your OpenAPI specification file using **x-kusk** [extension](/docs/extension.md).
+
+## Deploy services
+
+First, deploy the backend and frontend services:
+```
+kubectl apply -f examples/todomvc/backend
```

Comment:
this assumes that the reader must clone the repo. Better to have the full URLs of 2 files, or maybe even 1 file - combined frontend and backend.